src/Configuration/ConfigurationAnalyzer.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Qossmic\Deptrac\Configuration;
+
+final class ConfigurationAnalyzer
+{
+    /** @var array<string, mixed> */
+    private $config;
+
+    /**
+     * @param array<string, mixed> $arr
+     */
+    public static function fromArray(array $arr): self
+    {
+        return new self($arr);
```

Comment:
Can you please add an OptionsResolver here, like we have in `Qossmic\Deptrac\Configuration\Configuration`. The key `count_use_statements` should be required with bool type.

Reply:
Done in https://github.com/qossmic/deptrac/pull/571/commits/45ce6e08efd9ecaa4a722445c515b9bc65d62324